workers/ohsome_quality_analyst/indicators/ghs_pop_comparison_buildings/indicator.py

```diff
@@ -200,7 +199,6 @@ async def get_zonal_stats_population(self, bpolys: dict) -> Record:
                 st_setsrid(public.ST_GeomFromGeoJSON($3), 4326)
              )
             """
-        polygon = json.dumps(bpolys["features"][0]["geometry"])  # Geometry only
-        data = (polygon, polygon, polygon)
+        data = tuple(map(str, [self.feature.geometry] * 3))
```

Comment:
The pythonic way would be something like this:
```suggestion
        data = (str(self.feature.geometry),) * 3
```
At least using a list and a map seems a little bit unnecessary. If you want to stick with the list, you can do something like this:
```suggestion
        data = tuple([str(self.feature.geometry)] * 3)
```

Reply:
I implemented your last suggestion